lib/mockito.dart

```diff
@@ -404,3 +531,14 @@ void logInvocations(List<Mock> mocks) {
     print(inv.toString());
   });
 }
+
+/// Should only be used during Mockito testing.
```

Comment:
It's OK to be used outside. Not by users of the mockito package though. Perhaps by developers extending the mockito package.